apps/omg_watcher/lib/omg_watcher/sync_supervisor.ex

```diff
@@ -30,6 +30,15 @@ defmodule OMG.Watcher.SyncSupervisor do
   alias OMG.Watcher.Monitor
 
   @events_bucket :events_bucket
+  @status_cache :status_cache
+
+  def status_cache() do
+    @status_cache
+  end
+
+  def events_bucket() do
+    @events_bucket
+  end
```

Comment:
```suggestion
  def events_bucket(), do: @events_bucket
```

Reply:
you're here for the free commits aren't you